src/main.rs

```diff
@@ -48,6 +51,27 @@ enum Command {
 
     #[clap(subcommand)]
     Link(Link),
+
+    #[clap(subcommand)]
+    File(FileCommand),
+}
+
+#[derive(Subcommand, Debug)]
+enum FileCommand {
```

Comment:
We also need a read command. By default, it should return the latest version. The user can override version number and retrieve older content.

Would be awesome, for read and write commands, to handle jsons, i.e. allow the user modify particular fields easily, e.g. `ark-cli file write properties resource_id title "hello"`.